carsapi.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Gunicorn instance to serve carsapi
+After=network.target
+
+[Service]
+User=ubuntu
```

Comment:
Can you skip the `User` & `Group` directives here?
Using `ubuntu` as user will tie this service file to `Ubuntu` platform alone?

Reply:
Implemented the changes.